test/spec/detector_spec.js

```diff
@@ -27,19 +28,25 @@ describe('Detector', function() {
 
   describe('platform', function() {
 
-    let platform = null;
+    let platform = null,
+        device = null;
```

Comment:
Let's keep standard variable definitions as per line. I know why you're using it this way, but let's stick to original convention so coding standard doesn't vary as much across codebase. One line, one definition.